Added a save command to poll2.

ksmith0 opened the pull request:
The save command will write the DSP parameters to the set file specified
in pixie.cfg or provides an option to write to a different file for
backup.

ksmith0 committed:
Added a save command to poll2.

The save command will write the DSP parameters to the set file specified
in pixie.cfg or provides an option to write to a different file for
backup.

tking53 committed:
Fixed missing "

line 661 was missing an terminating ".

tking53 commented:
it was missing a terminating "; that my laptop didn't care about but Takk bailed during compilation

tking53 committed:
reworded help message; fixed expanded help

Previously save help would print the expanded help message but would also write out a file called help. Users wanted a bit more information on the brief help

tking53 reviewed (approved):
the save command works as expected. however the help wasn't detailed enough for Miguel. so i reworded that, and in the process of that i found that `save help` would print the expanded help message but would also save the parameters to a file called `help`. I fixed this. This was tested on takk
